core/src/wallet/common/synchronizers/AbstractBlockchainExplorerAccountSynchronizer.h

```diff
@@ -272,9 +272,9 @@ namespace ledger {
                                             std::shared_ptr<SynchronizationBuddy> buddy) {
 
                 buddy->logger->info("SYNC BATCHES");
-                //For ethereum like wallets, one account corresponds to one ETH address,
+                //For ethereum and ripple like wallets, one account corresponds to one ETH address,
                 //so ne need to discover other batches
-                auto isETHWallet = buddy->wallet->getWalletType() == api::WalletType::ETHEREUM;
+                auto hasMultipleAddresses = buddy->wallet->getWalletType() == api::WalletType::BITCOIN;
```

Comment:
I can see the complexity growing… 😨

Reply:
Actually, it's the opposite, we have networks/blockchains that will support multiple addresses/account (like UTXO based ones) and others having 1 address/account (like XRP, ETH etc..)